### PR TITLE
Ja-rule tweaks

### PR DESCRIPTION
**Pull request by peternewman**

### CODE REVIEW

File: tools/ja-rule/Makefile.mk [hunk 1]
```diff
@@ -11,6 +13,6 @@ tools_ja_rule_ja_rule_SOURCES = \
     tools/ja-rule/USBDeviceManager.h \
     tools/ja-rule/ja-rule.cpp
 tools_ja_rule_ja_rule_CXXFLAGS = $(COMMON_CXXFLAGS) $(libusb_CFLAGS)
-tools_ja_rule_ja_rule_LDADD = $(libusb_CFLAGS) \
+tools_ja_rule_ja_rule_LDADD = $(libusb_LIBS) \
```
Comment (peternewman) [inline tools/ja-rule/Makefile.mk]: I assume this is correct @nomis52 as it matches what we do in plugins/usbdmx

Comment (nomis52) [inline tools/ja-rule/Makefile.mk]: Yup.

Comment (peternewman) [inline tools/ja-rule/Makefile.mk]: Ah good.

**Comment by peternewman** [comment]
@nomis52 is the rest okay? I think Travis was only stuck incorrectly on Doxygen, but they're in maintenance now so I can't poke it.

**Comment by peternewman** [comment]
Probably nice to get into https://github.com/OpenLightingProject/ola/milestones/0.9.5 but not essential.

**Pull request merged**